src/main/java/com/opencart/datamodel/LoginModel.java

```diff
@@ -1,31 +1,63 @@
 package com.opencart.datamodel;
 
+
 public class LoginModel {
 
-    private final String email = "dfghszdfgh@gmail.com";
-    private final String newEmail = "newEmail@gmail.com";
-    private final String password = "dfghszdfgh";
-    private final String newPassword = "newPassword";
-    private final String newFirstName = "NewFirstName";
+    private String email = "dfghszdfgh@gmail.com";
+    private String newEmail = "newEmail@gmail.com";
+    private String password = "dfghszdfgh";
+    private String newPassword = "newPassword";
+    private String newFirstName = "NewFirstName";
+    private String wrongEmail = "uiu1@k.o";
+    private String wrongPassword = "12345678";
```

Comment:
You do not need to define those fields here. Set corresponding values directly into methods within LoginModelRepository class.
Leave only getEmail(), getPassword() getters in this class.

Reply:
Done